hs_core/management/commands/solr_recover.py

```diff
@@ -0,0 +1,125 @@
+"""This lists all the large resources and their statuses.
```

Comment:
This comment block looks to be left over from the command you copied.

Reply:
All comments fixed.